### DISCUSSION TITLE
Record lots of video on-camera, and download clips?

### DISCUSSION THREAD

**Discussion by JDLH** [AxisCommunications/discussions#14] [ACAP] [answered]
Hi, my boss has assigned me to start working on apps for our Axis camera. I'm new to everything but reading lots of documentation (thank you for it!). Our application is sort of surveillance or large outdoor event newscast. My boss has the impression that some Axis cameras can records lots (hours) of 4k video, then let our app ask for 30-60 second clips, so that the camera creates the video excerpt and points our app to it. Is this in fact possible?  What are the names of the functionality or SDK sections which I should look for, to learn more?

**Comment by marbali8** [comment, marked as answer]
Hi!

Thank you for reaching out. We discussed this and, if we understand this correctly, you first want to start a recording (we are guessing a continuous recording is the way to go here) to either an SD card (in the camera) or to a NAS (the cameras does not have any other built in storage space). Then your app needs to either play part of that recording directly, or download (i.e. export) a clip and then play that clip. This is absolutely doable.
So, basically, you first need to list the available recordings to find the correct Recording ID (you can choose to only list recordings within a given timeframe). And then after that play or export or play the proper clip. (Then it’s up to your application to take care of that clip.)
You can find all the relevant information in the VAPIX documentation under [Edge Storage API](https://www.axis.com/vapix-library/subjects/t10175981/section/t10004596/display). But specifically, you probably want to read about the [Recording API](https://www.axis.com/vapix-library/subjects/t10175981/section/t10004596/display?section=t10004596-t10004656) and the [Export recording API](https://www.axis.com/vapix-library/subjects/t10175981/section/t10004596/display?section=t10004596-t10047558).

**Reply by JDLH** [reply]
Thank you. This looks like exactly the sections I was looking for. This answers my question. I have some reading to do, and some experiments to try.
